README.md

```diff
@@ -46,6 +46,30 @@ st = currentProgram.symbolTable.getAllSymbols(1)
 for i in st:
     print('\n-----\nsymbol:' + str(i) + '\naddress:' + str(i.getAddress()))
 ```
+# How to manually generate the input files
```

Comment:
Tip: This should do it for now. keep in mind that once we find a way to do this in memory with the Ghidra API then we could ignore the whole manual labor here.

It is fine for now here

Reply:
I added here because I wanted to be sure that it is indeed what you were doing, can't improve if it isn't well defined.

Reply:
Sure thing. That comment was a FYI. All's good